### PR TITLE
Italian MAS tips translations

### PR DESCRIPTION
**Pull request by VinBoiSoft**
Update Localizable.strings (Italian)

### CODE REVIEW

**Comment by aonez** [comment]
Thanks Vincenzo!!!

File: Translations/it.lproj/Localizable.strings [hunk 1]
```diff
@@ -201,21 +201,21 @@ Note this will be used in Apple's servers
...
+"Monthly sip of coffee" = "Un sorso di caffè al mese";
+"Gift Keka a sip of coffee each month" = "Regala a Keka un sorso di caffè ogni mese";
+"Monthly coffee" = "Caffè mensile";
+"Gift Keka a coffee each month" = "Regala a Keka un caffè ogni mese";
+"Monthly pair of coffees" = "Un paio di caffè al mese";
+"Gift Keka a couple coffees each month" = "Regala a Keka un paio di caffè ogni mese";
+"Monthly breakfast" = "Colazione mensile";
+"Gift Keka a breakfast each month" = "Regala a Keka una colazione ogni mese";
+"Tip a coffee" = "Invita a bere un caffè";
+"Tip Keka a coffee so it can stay awake" = "Invita Keka a bere un caffè perché non si addormenti";
```
Comment (aonez) [inline Translations/it.lproj/Localizable.strings:213]: @VinBoiSoft there's a limit of 45 characters on the description and this string is 52, do you think you can cut it anywhere?

I'm using "Invita Keka a un caffè perché non si dorma" meanwhile.

File: Translations/it.lproj/Localizable.strings [hunk 2]
```diff
@@ -201,21 +201,21 @@ Note this will be used in Apple's servers
...
+"Tip a breakfast" = "Invita ad una colazione";
+"Tip Keka a breakfast so it can start fresh" = "Invita Keka a colazione per iniziare bene";
+"Tip a lunch" = "Regala un pranzo";
+"Tip Keka a lunch and feed it" = "Dai da mangiare a Keka per nutrirlo";
+"Tip a dinner" = "Regala una cena";
+"Tip Keka a dinner so it keeps rolling" = "Dai una cena a Keka perché continui a rotolare";
```
Comment (aonez) [inline Translations/it.lproj/Localizable.strings:219]: Same as above this string is 46, do you think you can cut it anywhere?

I'm using "Dai una cena a Keka perché continui a rotolar" meanwhile.

**Comment by VinBoiSoft** [comment]
@aonez See #688

**Pull request merged**